taxonomies/ingredients.txt

```diff
@@ -40782,6 +40788,14 @@ de:Ananassirup
 fr:sirop d'ananas
 nl:ananassiroop
 
+<en:pineapple
```

Comment:
We don't need to add entries for "dried something" or "freeze-dried something" as we have "dried" and "freeze-dried" in the ingredients_processing taxonomy.

So we already recognize "ananas séché" and "ananas lyophilisé" : https://fr.openfoodfacts.org/cgi/test_ingredients_analysis.pl?ingredients_text=ananas+s%C3%A9ch%C3%A9%2C+ananas+lyophilis%C3%A9&type=add&action=process&submit=Envoyer

Reply:
Ah cool! But it isn't working yet? I added those ingredients because they were not recognized

Reply:
It should be working, you can test with this tool: https://fr.openfoodfacts.org/cgi/test_ingredients_analysis.pl
We may not have reprocessed yet the ingredients of the products you saw.